modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -68,59 +67,69 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 +
 // Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
 
-. Convert the `journal.conf` file to base64:
+. Convert the `journal.conf` file to base64 and store it in a variable that is named `jrnl_cnf` by running the following command:
 +
 [source,terminal]
 ----
-$ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
+$ export jrnl_cnf=$( cat journald.conf | base64 -w0 )
 ----
 
-. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step.
-+
-For example:
+. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which was created in the previous step. The following sample command creates a MachineConfig for the  worker:
```

Comment:
With the last instance of MachineConfig in this instructional text, can you either enclose in backticks and include "object" like "`MachineConfig` object" or replace with "machine config".

For reference, the [API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) section of the docs guidelines.

If this sentence has been present all along and I failed to notice, please accept my apology.